tensorflow-core/tensorflow-core-generator/src/main/java/org/tensorflow/op/generator/ClassGenerator.java

```diff
@@ -1,19 +1,19 @@
 /*
-  Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Where's the actual change in this file? It looks like it's all formatting?

Reply:
I changed how the op builder is gotten, in `buildFactoryMethods`.